cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -945,6 +945,18 @@ typedef struct {
  */
 #define WDT_HAS_INIT                   (1)
 
+/**
+ * @brief   Frequency meter configuration
+ *
+ */
+#ifdef MODULE_PERIPH_FREQM
+struct freqm_config {
+    gpio_t msr_clk_src;     /*< GPIO at which the frequency is to be measured */
```

Comment:
```suggestion
    gpio_t pin;             /*< GPIO at which the frequency is to be measured */
```
We can use a more straightforward name here.